suite-common/wallet-core/src/device/initEvoluKeysThunk.ts

```diff
@@ -40,13 +40,18 @@ export const initEvoluKeysThunk = createThunk<void, InitCipherKeyThunkParams, vo
         );
 
         try {
+            // TODO: Implement proper proof_of_delegated_identity generation
+            // This should be a cryptographic proof that verifies the delegated identity
+            const proofOfDelegatedIdentity = new Uint8Array(0).buffer; // Placeholder - needs proper implementation
+
             const result = await TrezorConnect.evoluGetNode({
                 device: {
                     path: device.path,
                     state: device.state,
                     instance: device.instance,
                 },
                 useEmptyPassphrase: device.useEmptyPassphrase,
+                proof_of_delegated_identity: proofOfDelegatedIdentity,
```

Comment:
can I make it so taht this isn't snake case? Even though, that in Protobuf, it is defined as snake case?